packages/common/workspace/src/engine/blob.ts

```diff
@@ -78,6 +98,11 @@ export class BlobEngine {
               await remote.set(key, data);
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 76 to 76]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->We should add a condition here

```
if (this.status. isStorageOverCapacity) {
  return;
}
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5535?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->